src/main/java/reposense/parser/AuthorConfigCsvParser.java

```diff
@@ -103,31 +102,25 @@ private static AuthorConfiguration findMatchingAuthorConfiguration(
     /**
      * Associates {@code emails} to {@code author}, if provided and not empty.
      */
-    private static void setEmails(AuthorConfiguration config, Author author, List<String> emails) {
+    private static void setEmails(Author author, List<String> emails) {
         author.setEmails(new ArrayList<>(emails));
-        config.addAuthorEmailsAndAliasesMapEntry(author, author.getEmails());
     }
 
     /**
      * Associates {@code displayName} to {@code author}, if provided and not empty.
      * Otherwise, use github id from {@code author}.
      */
-    private static void setDisplayName(AuthorConfiguration config, Author author, String displayName) {
+    private static void setDisplayName(Author author, String displayName) {
         author.setDisplayName(!displayName.isEmpty() ? displayName : author.getGitId());
-        config.setAuthorDisplayName(author, !displayName.isEmpty() ? displayName : author.getGitId());
     }
 
     /**
-     * Associates {@code gitHubId} and additional {@code aliases} to {@code author}.
+     * Associates {@code aliases} to {@code author}.
      */
-    private static void setAliases(AuthorConfiguration config, Author author, String gitHubId, List<String> aliases) {
-        config.addAuthorEmailsAndAliasesMapEntry(author, Arrays.asList(gitHubId));
-
+    private static void setAliases(Author author, List<String> aliases) {
         if (aliases.isEmpty()) {
```

Comment:
Could you clarify the problem with the use of `return` ?

In my view, the existing style should be kept as it seems simpler to keep the common case outside the if-statement.

It is also more consistent with the style in other parts of the codebase.
Here is a similar example, in the `CsvParser` class:
https://github.com/reposense/RepoSense/blob/e24e9143dd136b2044eb82bf023465816182d3b5/src/main/java/reposense/parser/CsvParser.java#L164-L171

Reply:
There is no problem as such. I just think that since it's a `void` function we should avoid the use of return if possible. 

I feel the cases are a bit different. Since in the second case it needs to return a list. But, in the case we are discussing it doesn't need to. We can keep it the way it is, but if we can avoid using return, my personal opinion is that we should.

Reply:
Thanks for the reply. Can we keep it the way it is then?
Inverting the if-statement increases nesting, which seems less readable to me.
If we invert the if-statement, future programmers will also need to remember *not* to place code outside the if-statement.

Reply:
Sure! Let's proceed this way then.

Reply:
Great. Could I get your approval, if there are no other issues?